aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/connection-resiliency-and-command-interception-with-the-entity-framework-in-an-asp-net-mvc-application.md

```diff
@@ -1,28 +1,35 @@
 ---
 uid: mvc/overview/getting-started/getting-started-with-ef-using-mvc/connection-resiliency-and-command-interception-with-the-entity-framework-in-an-asp-net-mvc-application
-title: "Connection Resiliency and Command Interception with the Entity Framework in an ASP.NET MVC Application | Microsoft Docs"
+title: "Tutorial: Create connection resiliency and command interception with the EF in an ASP.NET MVC app"
```

Comment:
```suggestion
title: "Tutorial: Use connection resiliency and command interception with EF in an ASP.NET MVC app"
```

There's no single good verb to use with both these terms, but both refer to EF features, so Use seems better than Create. Another alternative is "Implement".